Reneiski_Homework1/ReneiskiSelenium11/PageObjects/Pages/RegFormPage.cs

```diff
@@ -0,0 +1,32 @@
+﻿using OpenQA.Selenium;
+using ReneiskiSelenium11.Common.WebElements;
+using ReneiskiSelenium11.Data;
+
+namespace ReneiskiSelenium11.PageObjects.Pages
+{
+    public class RegFormPage : WebTablePage
```

Comment:
some general comments on this page object:
1. Here should be no inheritance from WebTablePage because inheritance it's about common nature of objects (ex. Women inherits from Person). in this example, you just click button on WebTablePage and new popup openes
2. It's not actually a page, but a popup. Please create a new folder in PageObjects - "Popups" and put this PO in it

Reply:
Fixed